### ISSUE TITLE
QC data

### ISSUE THREAD

**Issue by hankman-siu** [pbaylis/vse-risk-tool#10]
Even though we aren't displaying QC in the Risk Tool dashboard, we should still have the QC csv data in the data/output folder, so it is easy for anyone to access!

**Comment by TonyBonen** [comment]
Agreed - and the QC data now has column names matching the other provinces I believe (thanks for that!).

But it appears that subsector_gdp_share is missing for Quebec. Can this be added to the CSV file for the NOC4 x NAICS3 data?
